inst/htmljs/animint.js

```diff
@@ -2055,45 +2055,55 @@ var animint = function (to_select, json_file) {
   find_maximum_dimensions()
   function generateKey(row, col) {
     return `${row},${col}`;
-}
-    var count_dimensions = Math.ceil(Math.sqrt(count))-1;
-    maximum_row=Math.max(maximum_row,count_dimensions)
-    maximum_column=Math.max(maximum_column,count_dimensions)
-    var table_width=(maximum_column+1)*500+"px"
-    var outer_table = element.append("table").attr("id", "outerTable").style("width", table_width).style("border", "1px solid black");
-    let span_map = new Map();
-    for (var p_name in response.plots) {
-       if (('rowspan' in response.plots[p_name].span)){
-     
-         let key = generateKey(response.plots[p_name].position.row, response.plots[p_name].position.col);
-         span_map.set(key,response.plots[p_name].span)
-         
-       }
-    }
     
+  }
+  var count_dimensions = Math.ceil(Math.sqrt(count))-1;
+  maximum_row=Math.max(maximum_row,count_dimensions)
+  maximum_column=Math.max(maximum_column,count_dimensions)
+  var table_width=(maximum_column+1)*500+"px"
```

Comment:
please remove magic number 500

Reply:
this number was removed. now no hardcoded values for cell dimensions